crates/loader/Cargo.toml

```diff
@@ -24,6 +24,8 @@ reqwest = "0.11.9"
 semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+# Brought in for FixedVersion
+spin-app = { path = "../app" }
```

Comment:
Should we duplicate the implementation rather than relying on the app crate?

SPIN UTILS NOW

Reply:
Yeah i agree duplication is better. Went that route instead